handler/src/test/java/io/netty/handler/flow/FlowControlHandlerTest.java

```diff
@@ -465,7 +465,7 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
         channel.register();
 
         // Reset read timeout by some message
-        assertTrue(channel.writeInbound(Unpooled.EMPTY_BUFFER));
+        assertFalse(channel.writeInbound(Unpooled.EMPTY_BUFFER));
```

Comment:
@carryxyh can you explain why these changes are needed ?

Reply:
Because when we set `autoread = false`, `writeInbound` will always return false.